dashboard/src/components/AppView/AccessURLItem.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react";
+
+import { IResource, IServiceSpec, IServiceStatus } from "../../shared/types";
+import "./AccessURLItem.css";
+
+interface IAccessURLItem {
+  service: IResource;
+}
+
+interface IAccessURLState {
+  isLink: boolean;
+  URLs: string[];
+}
+
+class AccessURLItem extends React.Component<IAccessURLItem, IAccessURLState> {
+  public state: IAccessURLState = {
```

Comment:
Do you need state? neither URL nor isLink seem to change so state is not needed no?